bundle.Dockerfile

```diff
@@ -15,6 +15,6 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 # Copy files to locations specified by labels.
-COPY bundle/manifests /manifests/
-COPY bundle/metadata /metadata/
-COPY bundle/tests/scorecard /tests/scorecard/
+COPY bundle/app/manifests /manifests/
```

Comment:
Why this change? Won't it break the operator-sdk scaffold compatibility?

Reply:
No because we updated the Makefile.

If we would like to have SonataFlow reference in the community CSV and OpenShift Serverless Logic into the productized CSV it is the only way to go.

Reply:
I restore the default folder structure for the community release